com.unity.render-pipelines.universal/Runtime/RenderingUtils.cs

```diff
@@ -254,6 +285,51 @@ internal static bool Contains(RenderTargetIdentifier[] source, RenderTargetIdent
             return false;
         }
 
+        //TODO: what should we do about Transient attachments? Currently can only exclude them from this.
+        /// <summary>
+        /// Return true if non-transient value can be found in source
+        /// </summary>
+        /// <param name="source"></param>
+        /// <param name="value"></param>
+        /// <returns></returns>
+        internal static bool Contains(AttachmentDescriptor[] source, AttachmentDescriptor value)
```

Comment:
This should have a different name if we really need this special handling of transient

Reply:
actually this transient check is not needed here anymore, so will remove it and update the comment